scripts/init-database.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python
+
+import yaml
+import sqlalchemy
+import os
+
+values = 'configurations'
+path = 'root_path'
+
+
+def retreive_values():
+    global path
+    path= os.path.abspath(os.path.join(os.path.join(__file__,os.pardir),os.pardir))
+    stream = open(path+"/bassa.yml", "r")
+    configs = yaml.load(stream)
+    global values
+    values = configs
+
+
+def create_database():
+    param = values['database']['database_type']+'://'+values['database']['database_user_username']+':'+values['database']['database_user_password']+'@'+values['database']['database_ip']
+    try:
+        engine = sqlalchemy.create_engine(param)
+	engine.execute("CREATE DATABASE IF NOT EXISTS " + values['database']['database_name'])
+    except:
+        param = values['database']['database_type']+'://root:'+values['database']['database_user_password']+'@'+values['database']['database_ip']
+	engine = sqlalchemy.create_engine(param)
+	engine.execute("CREATE DATABASE IF NOT EXISTS " + values['database']['database_name'])
+	engine.execute("GRANT ALL PRIVILEGES ON "+values['database']['database_name']+" TO "+values['database']['database_user_username']+"@"+values['database']['database_ip']+"WITH GRANT OPTION;")
```

Comment:
@padamchopra @malithsen We have to decide what are the privileges this new user should have?

I guess that is just, `INSERT`, `UPDATE`, `SELECT` and `DELETE`. If we move to user use soft deletes, we can drop `DELETE` also.